src/components/Card.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import {
+    Box,
+    Image,
+    Text,
+    Link,
+    Stack
+} from "@chakra-ui/react";
+
+interface Cardprops {
+    product: string
+    summary: string
+    longLine: string
+}
+
```

Comment:
Good use of typescript but are you sure this will be just strings? 

What I have in mind for example the `product` props can be something like this: 

```tsx
interface Product {
  name: string;
  // images can be an array of image soruces
  images: string[];
  price: number;
  description: string;
}
```

Do you remember the system design we did together? you can revisit it and check the `Product` schema to make sure its correct. 

This will be also in backend (we will do this together)

Reply:
My problem that i couldn't solve was the pics and how to call it back to product